devito/ops/transformer.py

```diff
@@ -35,16 +35,19 @@ def opsit(trees, count, name_to_ops_dat, block, dims):
         expressions.extend(FindNodes(Expression).visit(tree.inner))
 
     for expr in expressions:
-        ops_expressions.append(Expression(make_ops_ast(expr.expr, node_factory)))
+        ops_expressions.append(Expression(
```

Comment:
ops_expressions.extend([Expression(...) for expr in expressions])

Reply:
Fixed, now using python list comprehension. 

Also I'll add this same approach to the loop above this one.